R/plot_site_locations.R

```diff
@@ -21,7 +21,6 @@
 plot_site_locations <- function(traits, feature="trait_name", ...){
   
   au_map <- australia_map_raster %>%
-    raster::as.data.frame(xy = T) %>% 
```

Comment:
Is this not needed?

Reply:
No, since I just converted the raster into a data frame and saved that internally for the function to use! https://github.com/traitecoevo/austraits/blob/develop/data-raw/create_data.R 

I think we don't even need raster as a dependency for `plot_site_locations` to work now!